src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -107,6 +107,10 @@ module Types = struct
 
   let uint32 = uint32 ()
 
+  let block_time : (_, Block_time.t option) typ =
```

Comment:
Is this different than `Consensus.Data.Consensus_time.graphql_type`?

Reply:
Yes, This is a representation of `Core.Time.t`. `Consensus_time` has to do with slots and epochs